dev/com.ibm.ws.jsfContainer/src/com/ibm/ws/jsf/container/cdi/IBMViewHandlerProxy.java

```diff
@@ -33,6 +33,13 @@
 // class from weld at build-time and copy it here.  Then we can write a class that extends the CAVH.
 // This will allow us to automatically pick up any changes that are delivered into new versions of weld's CAVH
 
+//The reason we dpublicate a weld class here is becaus com.ibm.ws.jsfContainer is loaded by the app classloader
```

Comment:
```suggestion
//The reason we duplicate a weld class here is because com.ibm.ws.jsfContainer is loaded by the app classloader
```

Fixing up grammar a bit.

Otherwise, this comment looks good to me.